java/cdk/src/main/java/sleeper/cdk/stack/IngestStack.java

```diff
@@ -317,7 +320,7 @@ private void lambdaToCreateIngestTasks(IBucket configBucket, Queue ingestJobQueu
                 .create(this, "IngestTasksCreationPeriodicTrigger")
                 .ruleName(ruleName)
                 .description("A rule to periodically trigger the ingest tasks lambda")
-                .enabled(Boolean.TRUE)
+                .enabled(!deployPaused)
```

Comment:
I think this would work with `shouldDeployPaused(this)`, using the stack's context rather than the app's.